src/c++/perf_analyzer/report_writer.cc

```diff
@@ -388,4 +402,56 @@ ReportWriter::WriteGpuMetrics(std::ostream& ofs, const Metrics& metric)
   }
 }
 
+void
+ReportWriter::WriteLLMMetrics(std::ostream& ofs)
+{
+  auto [avg_first_token_latency, avg_t2t_latency] = CalculateLLMMetrics();
+
+  if (avg_first_token_latency.has_value()) {
+    ofs << "," << avg_first_token_latency.value();
+  } else {
+    ofs << ",n/a";
```

Comment:
I feel like N/A is normally capitalized?

Reply:
Fixed